### PR TITLE
upgraded to hapi17

### PR DESCRIPTION
**Pull request by bmleight**
based on @zemccartney's work here: https://github.com/BigRoomStudios/cupie-react/pull/9

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -96,11 +96,11 @@
...
     "fs-extra": "^3.0.1",
-    "hapi": "^16.1.1",
+    "hapi": "^17.0.0",
     "history": "^4.7.2",
     "html-webpack-plugin": "^2.7.1",
     "imports-loader": "^0.7.1",
-    "inert": "^4.0.0",
+    "inert": "^5.0.0",
     "keymirror": "^0.1.1",
     "labbable": "^2.1.1",
```
Comment (devinivy) [inline package.json]: Remove this lil one?

Comment (wswoodruff) [reply package.json]: Pour some out 🍷

Comment (bmleight) [reply package.json]: good catch.  All gone.

**Comment by zemccartney** [comment]
👏 🙏 🍷

**Comment by zemccartney** [comment]
i tried and failed to approve. :trollface: 
LGTM! Thanks for handling this, Bri 🙏

**Pull request merged**